packages/x-components/src/x-installer/api/api.types.ts

```diff
@@ -116,7 +116,7 @@ export interface SnippetConfig {
   /** The filters to be applied on the first request. */
   filters?: string[];
   /** List of queries to preview. */
-  queriesPreview?: QueryPreview[];
+  queriesPreview?: QueryPreviewConfig[];
```

Comment:
I think that this name its better for what its being used for, but I'm not 100% sure if i like it hahahaha

Reply:
What about `QueriesPreviewList`?

Reply:
I'm not sure about `QueriesPreviewList` as it should represent a single item instead of a list. What about `QueryPreviewDetails`, `QueryPreviewParams` or `QueryToPreview`?